#!/usr/bin/env python

PACKAGE = 'jsk_perception'

from dynamic_reconfigure.parameter_generator_catkin import *;

gen = ParameterGenerator ()

gen.add("filter_size", int_t, 0, "The diameter of each pixel neighborhood.", 7, 1, 50)
gen.add("sigma_color", double_t, 0, "Standard deviation in the color space.", 35, 0.0, 100)
gen.add("sigma_space", double_t, 0, "tandard deviation in the coordinate space (in pixel terms)", 5, 0.0, 100)

exit (gen.generate (PACKAGE, "jsk_perception", "BilateralFilter"))
